src/js/view/title.ts

```diff
@@ -47,9 +65,9 @@ Object.assign(Title.prototype, {
             }
         }
         style(this.el, titleStyle);
-    },
+    }
 
-    playlistItem: function(model, item) {
+    playlistItem(model: ViewModel, item: PlaylistMRSSItemWithMedia| PlaylistFeedItemWithMedia): void {
```

Comment:
We are listening to changes on the `PlayerViewModel`.

Reply:
`.player()` only exists on `ViewModel` with the way the classes are currently structured. the superclass of which is `PlayerViewModel` but it would have to be a `ViewModel` passed in here for the code to work as is written

Reply:
ah nice catch - ignore my previous comment, thought this was on the constructor